doc/tutorial/python/tutorial-install-python-bindings.dox

```diff
@@ -0,0 +1,703 @@
+  You need to activate it first though (in every `cmd` Command Prompt terminal instance where you are working on your project):
+
+      C:\> venv\Scripts\activate
+      (venv) C:\> mkdir visp-build-vc17-bindings
+      (venv) C:\> cd visp-build-vc17-bindings
+      (venv) C:\> cmake -G "Visual Studio 17 2022" -A "x64" -Dpybind11_DIR=C:\Users\%USERNAME%\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\pybind11\share\cmake\pybind11 ..\visp
+
+  At this point in `ViSP-third-party.txt` file you should see something similar to:
+
+      (venv) C:\> type ViSP-third-party.txt
+      ...
+      --   Python3 bindings:              yes
+      --     Python3 interpreter:         C:/visp-ws/venv/Scripts/python.exe (ver 3.12.2)
+      --     Pybind11:                    C:\Users\%USERNAME%\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\pybind11\share\cmake\pybind11 (2.11.1)
+      --     Package version:             3.6.1
+      --     Wrapped modules:             core dnn_tracker gui imgproc io klt me sensor ar blob robot visual_features vs vision detection mbt tt tt_mi
+      --     Generated input config:      C:/visp-ws/visp-build-vc17-bindings/modules/python/cmake_config.json
+
+- Build Python bindings
+
+      (venv) C:\> cmake --build . --config Release --target visp_python_bindings
+
+- Build Python bindings documentation
+
+      (venv) C:\> cmake --build . --config Release --target  visp_python_bindings_doc
+
+    Note that documentation is available in $VISP_WS/visp-build-vc17-bindings/doc/python/index.html
+
+
+\subsection py_bindings_build_conda Using conda
+
+\subsubsection py_bindings_build_conda_macos How to build on macOS
+
+- If not already done, install [Miniforge](https://github.com/conda-forge/miniforge)
+
+      $ wget https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-arm64.sh -O /tmp/Miniforge3-MacOSX-arm64.sh
+      $ zsh /tmp/Miniforge3-MacOSX-arm64.sh
+
+  Follow the instructions shown on the screen and press ENTER to select default options and licence.
+
+      You can undo this by running `conda init --reverse $SHELL`? [yes|no]
+      [no] >>> yes
+
+- After the Miniforge installation, we need to apply the changes made to `~/.zshrc` file.
+  Miniconda installer modified the file during the installation.
+
+      $ source ~/.zshrc
+
+- Check installation by retrieving Conda version
+
+      (base) $ conda info
+      ...
+      conda version : 23.11.0
+      ...
+
+- Create a Conda environment
+
+      (base) $ conda create -n visp-conda-ws
+
+      Proceed ([y]/n)? y
+
+- Activate the Conda environment
+
+      (base) $ conda activate visp-conda-ws
+      (visp-conda-ws) $
+
+- Install pybind11 using conda
+
+      (visp-conda-ws) $ conda install pybind11
+
+      Proceed ([y]/n)? y
+
```

Comment:
We should probably tell the user that he can install Visp other dependencies here, even if they are optional for compiling Visp. This would solve the missing X11 library error we encounter just next, so at least this one should be installed.
May be something like:
```
- At this point, you can also install ViSP other dependencies using conda depending on which features you want to be compiled. You will need to install at least the X11 libraries:
     (visp-conda-ws) $ conda install xorg-libx11
  
  However, we recommend this minimal set of dependencies to get the main features of ViSP available:
     (visp-conda-ws) $ conda install xorg-libx11 eigen libopencv libjpeg-turbo libpng
```